buildbot/master/files/config/factories.py

```diff
@@ -67,8 +67,8 @@ def __init__(self, build_steps):
 
 class StepsYAMLParsingStep(buildstep.ShellMixin, buildstep.BuildStep):
     """\
-    Step which reads the YAML steps configuration in the main servo repo
-    and dynamically adds test steps.
+    Step which reads the YAML steps configuration in the repo and dynamically
```

Comment:
`the YAML steps` -> `a YAML steps`
I'd also keep the second part of the sentence (`and dynamically...`) on the next line.